### DISCUSSION TITLE
Multiple Monitor 'Breakout'

### DISCUSSION THREAD

**Discussion by modem-19** [amidaware/tacticalrmm#2024] [General]
When I remote into my office PC (2 monitors attached) is it possible to have the TacticalRMM remote window display two separate RMM windows for dual monitor support when I'm at home?

For example, when I remote into my office PC from home, I get a single Tactical RMM window that is shrunk with 2 screens showing within that single window.  I can click the monitors icon in the bottom right to select which remote monitor I want to focus on.  However, I would really like to know if there is the ability to have RMM separate into two separate windows showing my two remote monitors at the office.

Is this possible?

Thanks!

**Comment by dinger1986** [comment]
Not using mesh natively but the great thing is with tactical you can use other remote control solutions which would support this via a url action

**Reply by modem-19** [reply]
That's interesting, what other remote control solutions are supported with Tactical and are they natively implemented already?

**Reply by dinger1986** [reply]
Best to look at 3rd party integrations on our docs https://docs.tacticalrmm.com/. They use scripts and url actions.
